src/main/java/org/jboss/remoting3/util/InetUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.remoting3.util;
+
+import org.wildfly.common.net.Inet;
+
+/**
+ * @author baranowb
+ */
+public final class InetUtils {
```

Comment:
This is not API so it should not be public.

Reply:
https://github.com/jboss-remoting/jboss-remoting/blob/master/src/main/java/org/jboss/remoting3/util/StreamUtils.java#L32  ? Should it just be present in package as default level acccess?

Reply:
Yes, package-private is appropriate.